src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -42,6 +45,13 @@ var (
 )
 
 func init() {
+	if version == "" {
```

Comment:
I moved this block to here to make sure `version` and `buildDate` are available for use in `doMain`.